utils/build/docker/php/install_ddtrace.sh

```diff
@@ -2,25 +2,37 @@
 
```

Comment:
The requirement for this script : 

1. if it found some binaries inside /binaries folder then it install it
2. else, it downlaod the very last version of the tracer.
3. then, it extract the version of the tracer with whatever dark magic, and save it in a file named /binaries/SYSTEM_TESTS_LIBRARY_VERSION

From what I see, it starts by step 2, am I right ?

Reply:
I'm installing two things here: the tracer and appsec. What's stored in SYSTEM_TESTS_LIBRARY_VERSION is the version of the tracer, not the version of appsec. It doesn't support installing the latest version though, but that support should go into the installer that I'm using, not the script, as it's planned to include such support in the script. Right now, it installs 0.67.0 of the tracer if there's nothing in /binaries and fails if there's no appsec in /binaries. This is because the appsec repository is not yet public.

Reply:
> What's stored in SYSTEM_TESTS_LIBRARY_VERSION is the version of the appsec, not the version of the tracer

It must be the tracer version. If you need to use the `appsec` version to do some filters with `irrelevant`, `bug`... we need to add it. Tell me, and I will do it.

----

Ok, from what I understand, you look for a `.tar.gz` in `/binaries`, and it was previously a `.apk` right ? If yes, you also need to modify the `./utils/scripts/load-binary.sh` script => it's used in my nightly CI to get the latest build from your `master` branch.

Reply:
The version of the tracer is mostly irrelevant, since its only role is submitting the trace. For the functionality that we're testing, its version is not nearly as important as the version of the appsec library. So having `SYSTEM_TESTS_LIBRARY_VERSION` be the tracer version may make sense in other runtimes where the two integrated, but no so much in PHP. In any case, if you want `SYSTEM_TESTS_LIBRARY_VERSION` to be the tracer version, then yes, another mechanism will be needed to test against the appsec version.